### DISCUSSION TITLE
How i use drawstring in Cosmos user kit 20220209

### DISCUSSION THREAD

**Discussion by Samma2009** [CosmosOS/Cosmos#2296] [Q&A]
I Need help with the function `canvas.drawstring` can you help me?

**Comment by LoxyMono** [comment]
If I were you, I would use Nifanfa's DrawString code, since it draws much much faster than the built in function.

**Comment by LoxyMono** [comment]
https://github.com/nifanfa/Cosmos-DrawString/blob/master/ASC16.cs

**Comment by quajak** [comment]
What is your issue with the DrawString method?
